fastlane/Fastfile

```diff
@@ -39,19 +39,7 @@ EXTENSIONS_BUNDLE_IDENTIFIERS = %w[
   WordPressThisWeekWidget
   WordPressIntents
 ].map { |suffix| "#{APP_STORE_VERSION_BUNDLE_IDENTIFIER}.#{suffix}" }
-# rubocop:disable Style/MutableConstant
-#
-# Fastlane `match` manipulates the array of bundle identifiers it's given in a
-# way that expects it to be mutable. We cannot call `.freeze`, or `match` will
-# fail.
-#
-# See:
-#
-# - https://github.com/fastlane/fastlane/issues/11324
-# - https://github.com/fastlane/fastlane/blob/b817148d4967ab6e38b8f1a8b81d1b3e60d6e645/match/lib/match/runner.rb#L84-L86
-# - https://buildkite.com/automattic/wordpress-ios/builds/6240#ad789b9f-b744-4867-8d58-95921c58a3cf/2491-2756
```

Comment:
Fixed by https://github.com/fastlane/fastlane/pull/20125 since, which just landed in `2.205.2`.